modules/puppet_proxy_puppet_api/v3_environmentclassesapi_classes_retriever.rb

```diff
@@ -0,0 +1,100 @@
+require 'concurrent'
+
+class Proxy::PuppetApi::V3EnvironmentClassesApiClassesRetriever
```

Comment:
The filename is missing some underscores to match the capitalisation of the class, it'd be easier to read with them.